src/puppetlabs/trapperkeeper/internal.clj

```diff
@@ -135,16 +136,14 @@
       (handle-prismatic-exception! e))))
 
 
-(defn parse-cli-args!
+(schema/defn parse-cli-args! :- common/CLIData
   "Parses the command-line arguments using `puppetlabs.kitchensink.core/cli!`.
   Hard-codes the command-line arguments expected by trapperkeeper to be:
       --debug
       --bootstrap-config <bootstrap file>
       --config <.ini file or directory>
       --plugins <plugins directory>"
-  [cli-args]
-  {:pre  [(sequential? cli-args)]
-   :post [(map? %)]}
+  [cli-args :- [(schema/maybe schema/Str)]]
```

Comment:
Could this just be...

```clj
[cli-args :- [schema/Str]]
```
?

Wasn't sure if it was expected / possible for one of the arguments to be nil.